docs/docsite/rst/community/collection_contributors/collection_release_without_branches.rst

```diff
@@ -102,14 +102,15 @@ Publish the collection
     git tag -a NEW_VERSION -m "comment here"    # the comment can be, for example,  "community.postgresql: 1.2.0"
     git push upstream NEW_VERSION
 
-
+  .. note:: Make sure the release tag is in the format X.Y.Z - Zuul will not trigger the release process otherwise (as of Dec 2025).
 
 2. Wait until the new version is published on the collection's `Ansible Galaxy <https://galaxy.ansible.com/>`_ page. It will appear in a list of tarballs available to download.
 
 3. Update the version in the ``galaxy.yml`` file to the next **expected** version. Add, commit, and push to the ``upstream``'s default branch.
 
 4. Add a GitHub release for the new tag. Title should be the version and content ``See https://github.com/ansible-collections/community.xxx/blob/main/CHANGELOG.rst for all changes``.
 
-5. Announce the release through the `Bullhorn Newsletter issue <https://forum.ansible.com/c/news/bullhorn/17>`_.
+5. Announce the release in the ``#social`` :ref:`Matrix/IRC channel <communication_irc>`.
+   By mentioning ``@newsbot`` it will automatically go into the next edition of the `Bullhorn Newsletter issue <https://forum.ansible.com/c/news/bullhorn/>`_.
 
-6. Announce the release in the pinned release issue/community pinboard of the collection mentioned in step 3 and in the ``community`` :ref:`Matrix/IRC channel <communication_irc>`.
+6. Announce the release in the pinned release issue/community pinboard of the collection mentioned in step 3 if one is being used.
```

Comment:
We should be more precise here. "Step 3" referes to another list in this file, not to the current list (where item 3 is about galaxy.yml).

Reply:
Indeed, thanks! Reworded